backend/package.json

```diff
@@ -5,11 +5,12 @@
   "main": "src/app.ts",
   "type": "module",
   "scripts": {
-    "dev": "tsc && node --watch dist/app.js",
```

Comment:
Does this remove hot reloading from the config? Live debugging has been very nice so far

Reply:
Was it working for you on the backend?

Reply:
Technically tsc is just running once so even if you changed the typescript, dist/app/js wouldn't update yet.

I tried playing around with a couple other options for typescript but I didn't have any luck

Reply:
Except actually I think I just thought of exactly what was wrong typing that standby

Reply:
Alright yeah that did it. HMR is actually working properly on the backend and will recompile and restart the server anytime you save changes!

Reply:
> Was it working for you on the backend?

Errr you're right. It wasn't working for changes on the backend (took me way to long to realize that!) but for frontend it is very nice.

> Alright yeah that did it. HMR is actually working properly on the backend and will recompile and restart the server anytime you save changes!

Sweet! Thanks for adding that!